formats/calendar/resources/ext.srf.formats.eventcalendar.js

```diff
@@ -467,6 +471,30 @@
 							// If the day number (where available) is clicked then switch to the daily view
 							if ( allDay && data.query.ask.parameters.dayview && $( jsEvent.target ).is( 'div.fc-day-number' ) ) {
 								container.fullCalendar( 'changeView', 'agendaDay'/* or 'basicDay' */).fullCalendar( 'gotoDate', date );
+							} else {
```

Comment:
Hello  mwjames,
Thanks for his suggestion about the function. 
I am  about to rework a few things and will let you people know, when the Pull Request is ready to be merged.